shared/js/initApp.js

```diff
@@ -1,9 +1,21 @@
+// TODO (brent) - way too many globals
+// TODO (brent) - I wonder if we should sub-namespace dashboard
+/* global script_path, Dialog, CDOSounds, dashboard, appOptions, $, trackEvent, Blockly, Applab, sendReport, cancelReport, lastServerResponse, showVideoDialog, ga*/
```

Comment:
@joshlory If you're going to look at one part of this, this is probably the right part. Also, once this is in, I wonder if we can extra some pieces of logic from this file as right now it feels fairly monolithic.